includes/class-wc-swedbank-plugin.php

```diff
@@ -0,0 +1,554 @@
+<?php
+
+namespace SwedbankPay\Payments\WooCommerce;
+
+defined( 'ABSPATH' ) || exit;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of exit language construct is discouraged.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)